packages/elements/src/item/__test__/item.test.js

```diff
@@ -1,8 +1,33 @@
-import { elementUpdated, expect, fixture, nextFrame } from '@refinitiv-ui/test-helpers';
```

Comment:
Do you really need to add the real test? Essentially you create tight coupling between `item` and `tooltip`. Whenever tooltip changes, this test would stop working.

Maybe it is better to test just the functionality  of `getItemContent` and `isItemOverflown` functions? You can make those protected btw.

Reply:
Updated.